### PR TITLE
feat(global): add landing page

### PR DESCRIPTION
**Pull request by velrest**
image

### CODE REVIEW

**Comment by velrest** [comment]
image
I added a coming soon message in case we have to deploy quarterly closure before anything else. Just in case.

File: addon/routes/application.js [hunk 1]
```diff
@@ -1,3 +1,9 @@
 import Route from "@ember/routing/route";
 
-export default class ApplicationRoute extends Route {}
+export default class ApplicationRoute extends Route {
+  afterModel({ id } = {}) {
+    if (typeof id === "string") {
```
Comment (luytena) [inline addon/routes/application.js:5]: If it isn't in the context of project then it is `undefined`?

File: addon/routes/building/new.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ export default class BuildingNewRoute extends BuildingFormRoute {
 
   async model() {
     const model = {
-      instanceId: this.modelFor("application").id,
+      instanceId: this.modelFor("application")?.id,
```
Comment (luytena) [inline addon/routes/building/new.js:21]: Will the `instanceId` ever be undefined here?

File: addon/routes/project/new.js [hunk 3]
```diff
@@ -19,6 +19,6 @@ export default class ProjectNewRoute extends FormRoute {
       this.config.cantonAbbreviation;
     project.constructionSurveyDeptNumber =
       this.constructionProject.constructionSurveyDeptNumber;
-    return { project, instanceId: this.modelFor("application").id };
+    return { project, instanceId: this.modelFor("application")?.id };
```
Comment (luytena) [inline addon/routes/project/new.js:22]: Will the `instanceId` ever be undefined here?

File: addon/routes/application.js (hunk 1 above)
Comment (velrest) [reply addon/routes/application.js:5]: Yes exactly

File: addon/routes/building/new.js (hunk 2 above)
Comment (velrest) [reply addon/routes/building/new.js:21]: maybe not but i added the optional chaining everywhere and it never hurts to be a bit pessimistic

File: addon/routes/project/new.js (hunk 3 above)
Comment (velrest) [reply addon/routes/project/new.js:22]: same as above

File: addon/templates/landing-page.hbs [hunk 4]
```diff
@@ -0,0 +1,64 @@
+<div class="uk-padding">
```
Comment (luytena) [inline addon/templates/landing-page.hbs:1]: Delete

File: addon/templates/global-landing-page.hbs [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  <h1 class="uk-text-center">
+    {{t "ember-gwr.globalLandingPage.heading"}}
+  </h1>
+  <h4 class="uk-text-center uk-margin-remove-top uk-margin-large">
+    {{t "ember-gwr.globalLandingPage.infoText"}}
+  </h4>
+
+  <div uk-grid class="uk-child-width-1-1 uk-child-width-1-3@s">
+    <UkFlex @horizontal="center" class="uk-flex-right@s">
+      <LinkTo
```
Comment (luytena) [inline addon/templates/global-landing-page.hbs:11]: Make component out of tiles

File: addon/templates/global-landing-page.hbs [hunk 6]
```diff
@@ -0,0 +1,70 @@
+<div class="uk-padding">
+  <h1 class="uk-text-center">
+    {{t "ember-gwr.globalLandingPage.heading"}}
+  </h1>
+  <h4 class="uk-text-center uk-margin-remove-top uk-margin-large">
+    {{t "ember-gwr.globalLandingPage.infoText"}}
+  </h4>
+
+  <div uk-grid class="uk-child-width-1-1 uk-child-width-1-3@s">
```
Comment (luytena) [inline addon/templates/global-landing-page.hbs:9]: Test with small width

File: translations/fr.yaml [hunk 7]
```diff
@@ -44,12 +45,19 @@ ember-gwr:
...
     createNewProject: Saisir un nouveau projet
     linkExistingProject: Lier un projet existant
 
+  globalLandingPage:
+    heading: GWR
+    infoText: Le module GWR permet de rechercher des projets et des bâtiments existants dans le GWR et de procéder à des adaptations. De plus, la clôture trimestrielle peut être effectuée. Les projets de construction GWR qui sont déjà liés à un dossier eBau peuvent également être consultés sous le dossier eBau correspondant.
+    searchProject: Rechercher et traiter des projets de construction
+    searchBuilding: Rechercher et traiter des bâtiments
+    quarterlyClosure: Clôture trimestrielle
+
```
Comment (luytena) [inline translations/fr.yaml:60]: Replace with empty strings

File: addon/templates/global-landing-page.hbs (hunk 6 above)
Comment (velrest) [reply addon/templates/global-landing-page.hbs]: Works, if the screen is reeeeally small they just get stacked atop one another.

File: addon/templates/landing-page.hbs (hunk 4 above)
Comment (velrest) [reply addon/templates/landing-page.hbs]: Done

File: translations/fr.yaml (hunk 7 above)
Comment (velrest) [reply translations/fr.yaml]: Done

**Review by luytena — APPROVED** [review]

**Review by luytena — APPROVED** [review]

**Pull request merged**